pandas/tests/indexes/timedeltas/test_ops.py

```diff
@@ -334,7 +334,7 @@ def test_freq_setter_errors(self):
             idx.freq = '5D'
 
         # setting with a non-fixed frequency
-        msg = '<2 \* BusinessDays> is a non-fixed frequency'
+        msg = r'<2 \* BusinessDays> is a non-fixed frequency'
```

Comment:
Hmm what failure was this causing? Change certainly looks correct just not sure how it's related to warnings

Reply:
Python 3.something (6ish?) added DeprecationWarnings for invalid escape sequences in strings.

```
bash-4.4$ cat bad.py
# file: bad.py
x = "<2 \* Day>"
bash-4.4$ python3 -W error bad.py
  File "bad.py", line 2
    x = "<2 \* Day>"
       ^
SyntaxError: invalid escape sequence \*
```

Reply:
Was CI not hitting this before this change?

Reply:
It was, e.g. https://travis-ci.org/pandas-dev/pandas/jobs/429647889#L2261